packages/core/test/unit/test-runner/CommandTestRunner.spec.ts

```diff
@@ -37,6 +37,14 @@ describe(CommandTestRunner.name, () => {
       expect(childProcess.exec).calledWith('some other command');
     });
 
+    it('should allow other commands using function-based configuration', async () => {
+      const command = sinon.fake.returns('some functional command');
```

Comment:
Out of curiosity, why create a sinon `fake` here? I've only worked with `stub()`. What is the difference exactly?

Reply:
I don't know what the difference is. I'm not familiar with sinon, so I had I to dig into sinon's docs when I saw that's what you're using. I can change it to stub().